composer.json

```diff
@@ -23,5 +23,8 @@
         "mozilla/wp-sw-manager": "dev-master",
         "marco-c/wp_serve_file": "dev-master"
     },
+    "autoload": {
+      "classmap": ["vendor/"]
```

Comment:
If I do this and https://github.com/marco-c/wp-web-app-manifest-generator/pull/9 and run `composer autoload`, the generated `autoload_classmap.php` is empty.